src/translations/pages/departures.ts

```diff
@@ -77,5 +89,6 @@ export const Departures = {
         'Sjå fleire avgangar',
       ),
     },
+    noSearchResults: {},
```

Comment:
Unused?

Reply:
Yes, edited to `emptySearchResults` and forgot to remove it!